### ISSUE TITLE
[FEATURE] Improve templates page accessibility

### ISSUE THREAD

**Issue by Olaleye-Blessing** [formbricks/formbricks#1572]
### Is your feature request related to a problem? Please describe.

The survey templates page lacks basic accessibility. Keyboard users will have a hard time making use of the page cause templates are not focusable due to the wrong use of HTML tags. 

[Loom Video](https://www.loom.com/share/c9e233ea84854978aed83cc49c759f15?sid=230d39c8-2c61-4c6a-99a0-195a2c1c0dbf)

Generally, most of the divs can be changed to use right HTML tags from the search field, to the list of categories, to the list of templates.

<img width="1512" alt="Screenshot 2023-11-03 at 18 15 24" src="https://github.com/formbricks/formbricks/assets/70102539/ed543c7e-988b-4583-9023-4afe3d630f94">

### Describe the solution you'd like

At the least, the templates should be focusable.

### Describe alternatives you've considered

The whole page should be accessible to some extent not just to mouse user.

### Additional context

I will like to work on it.

**Comment by mattinannt** [comment]
@Olaleye-Blessing thanks a lot for pointing this out. I will add the "help wanted" label to see if one of the community contributors would like to take this up :-)

**Comment by ksushant6566** [comment]
Hi @mattinannt , I can take this up :)

**Comment by ksushant6566** [comment]
Hi @mattinannt can you confirm if this still needs to be done, if yes please assign it to me?

**Comment by jobenjada** [comment]
hey @ksushant6566 - sorry this fell through the cracks! :) Please have a go, would love to get your contribution merged asap :)

**Comment by ksushant6566** [comment]
Hey @jobenjada, No worries. I'll get to it and keep you guys posted.
